<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2024, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorExportForm</class>
 <widget class="QDialog" name="qtractorExportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>240</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtractor.qrc">:/images/qtractor.svg</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="ExportPathTextLabel">
     <property name="text">
      <string>&amp;File:</string>
     </property>
     <property name="buddy">
      <cstring>ExportPathComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="ExportPathComboBox">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Export file name</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ExportPathToolButton">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Browse export file name</string>
       </property>
       <property name="icon">
        <iconset theme="fileOpen" />
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QStackedWidget" name="ExportTypeWidget">
     <widget class="QWidget" name="AudioExportTypePage">
      <layout class="QGridLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="AudioExportTypeTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignVCenter</set>
         </property>
         <property name="text">
          <string>File &amp;type:</string>
         </property>
         <property name="buddy">
          <cstring>AudioExportTypeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="qtractorAudioFileTypeComboBox" name="AudioExportTypeComboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Audio file type to use on export</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="AudioExportFormatTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignVCenter</set>
         </property>
         <property name="text">
          <string>Sample &amp;format:</string>
         </property>
         <property name="buddy">
          <cstring>AudioExportFormatComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="qtractorAudioFileFormatComboBox" name="AudioExportFormatComboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Audio sample format to use on export</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="AudioExportQualityTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignVCenter</set>
         </property>
         <property name="text">
          <string>&amp;Quality:</string>
         </property>
         <property name="buddy">
          <cstring>AudioExportQualitySpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="AudioExportQualitySpinBox">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>32767</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Audio compression quality to use on export</string>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MidiExportTypePage">
      <layout class="QGridLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="MidiExportFormatTextLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignVCenter</set>
         </property>
         <property name="text">
          <string>File &amp;format:</string>
         </property>
         <property name="buddy">
          <cstring>MidiExportFormatComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="qtractorMidiFileFormatComboBox" name="MidiExportFormatComboBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>MIDI file format to use on export</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="ExportRangeGroupBox">
     <property name="title">
      <string>Range</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0" colspan="3">
       <widget class="QRadioButton" name="SessionRangeRadioButton">
        <property name="toolTip">
         <string>Session range</string>
        </property>
        <property name="text">
         <string>&amp;Session</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QRadioButton" name="LoopRangeRadioButton">
        <property name="toolTip">
         <string>Loop range</string>
        </property>
        <property name="text">
         <string>&amp;Loop</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QRadioButton" name="PunchRangeRadioButton">
        <property name="toolTip">
         <string>Punch range</string>
        </property>
        <property name="text">
         <string>&amp;Punch</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QRadioButton" name="EditRangeRadioButton">
        <property name="toolTip">
         <string>Edit range</string>
        </property>
        <property name="text">
         <string>&amp;Edit</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QRadioButton" name="CustomRangeRadioButton">
        <property name="toolTip">
         <string>Custom range</string>
        </property>
        <property name="text">
         <string>&amp;Custom</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="ExportStartTextLLabel">
        <property name="text">
         <string>St&amp;art:</string>
        </property>
        <property name="buddy">
         <cstring>ExportStartSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="qtractorTimeSpinBox" name="ExportStartSpinBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Custom start</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="ExportEndTextLLabel">
        <property name="text">
         <string>En&amp;d:</string>
        </property>
        <property name="buddy">
         <cstring>ExportEndSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="qtractorTimeSpinBox" name="ExportEndSpinBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Custom end</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="7">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="ExportBusGroupBox">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Outputs</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QListWidget" name="ExportBusNameListBox">
        <property name="toolTip">
         <string>Output bus names</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="FormatGroupBox">
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QComboBox" name="FormatComboBox">
        <property name="toolTip">
         <string>Time display format</string>
        </property>
        <item>
         <property name="text">
          <string>Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Time</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BBT</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>8</number>
     </property>
     <item>
      <widget class="QCheckBox" name="AddTrackCheckBox">
       <property name="toolTip">
        <string>Whether to add/import new track(s) with export result</string>
       </property>
       <property name="text">
        <string>&amp;Add new track(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="DialogButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8"/>
 <customwidgets>
  <customwidget>
   <class>qtractorTimeSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qtractorSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>qtractorAudioFileFormatComboBox</class>
   <extends>QComboBox</extends>
   <header>qtractorComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qtractorAudioFileTypeComboBox</class>
   <extends>QComboBox</extends>
   <header>qtractorComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qtractorMidiFileFormatComboBox</class>
   <extends>QComboBox</extends>
   <header>qtractorComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ExportPathComboBox</tabstop>
  <tabstop>ExportPathToolButton</tabstop>
  <tabstop>AudioExportTypeComboBox</tabstop>
  <tabstop>AudioExportFormatComboBox</tabstop>
  <tabstop>AudioExportQualitySpinBox</tabstop>
  <tabstop>MidiExportFormatComboBox</tabstop>
  <tabstop>SessionRangeRadioButton</tabstop>
  <tabstop>LoopRangeRadioButton</tabstop>
  <tabstop>PunchRangeRadioButton</tabstop>
  <tabstop>EditRangeRadioButton</tabstop>
  <tabstop>CustomRangeRadioButton</tabstop>
  <tabstop>ExportStartSpinBox</tabstop>
  <tabstop>ExportEndSpinBox</tabstop>
  <tabstop>ExportBusNameListBox</tabstop>
  <tabstop>FormatComboBox</tabstop>
  <tabstop>AddTrackCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc"/>
 </resources>
 <connections/>
</ui>
